src/pages/BookTicket.vue

```diff
@@ -1,19 +1,22 @@
 <script setup lang="ts">
-import { onMounted, ref } from "vue";
+import { onMounted, ref, Ref } from "vue";
 import { storeToRefs } from "pinia";
 import { useReservationStore } from "@/stores/reservation.js";
 import SectionContainer from "@/components/global/SectionContainer.vue";
-import BookTicketsSteps from "@/components/global/BookTicketsSteps.vue";
+import BookTicketsSteps from "@/components/booking/BookTicketsSteps.vue";
 import SectionTitleSecondary from "@/components/global/SectionTitleSecondary.vue";
 import LoadingData from "@/components/global/LoadingData.vue";
 import ErrorMessage from "@/components/global/ErrorMessage.vue";
 import ScreeningMovieCard from "@/components/screenings/ScreeningMovieCard.vue";
+import ChooseSeats from "@/components/booking/ChooseSeats.vue";
+import ChooseTickets from "@/components/booking/ChooseTickets.vue";
+import BookingSuccess from "@/components/booking/BookingSuccess.vue";
 
 const props = defineProps<{
   movieSeanceId: String;
 }>();
 
-const step = ref(1);
+const step: Ref<number> = ref(1);
```

Comment:
TypeScript should infer that one properly 🤔 
```suggestion
const step = ref(1);
```